twitterstatus.py

```diff
@@ -2,12 +2,12 @@
 
 import json
 from twitter import Twitter, OAuth, TwitterHTTPError
-from time    import sleep
-from sys     import exit
-from random  import choice
+from time import sleep
+from sys import exit
+from random import choice
 
 try:
-    from config  import *
+    import config
```

Comment:
This would break everything below that actually uses variables from config.* so these would need to be changed as well

Reply:
I changed the rest of instances to use config.whatever. Having * is never a good practice.

Reply:
Not having * is nice but there's still variables like `LANGUAGE` and `OAUTH_*` floating around - those should be fixed as well

Reply:
Still in the new push?

Reply:
Okay, no, my bad. Apparently github was a little slow to report on that. The config stuff looks fine now :)